apps/src/templates/sectionProgressV2/IconKey.jsx

```diff
@@ -35,7 +34,7 @@ export default function IconKey({isViewingValidatedLevel, expandedLessonIds}) {
       aria-expanded={isOpen}
       aria-label={i18n.iconKey()}
     >
-      <div onClick={clickListener} className={styles.iconKeyTitle}>
+      <div onClick={clickListener} className={styles.iconKeyTitle} data-testid="expandable-container">
```

Comment:
I wanted to be able to target this `div` for clicking which is why I added in this `testid` here.